readme.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Add some information here, like instructions for installing/running the app and a description of the API methods.